src/server/scripts/Spells/spell_dh.cpp

```diff
@@ -16,11 +16,157 @@
  */
 
 /*
- * Scripts for spells with SPELLFAMILY_DEMONHUNTER and SPELLFAMILY_GENERIC spells used by deathknight players.
+ * Scripts for spells with SPELLFAMILY_DEMONHUNTER and SPELLFAMILY_GENERIC spells used by demon hunter players.
  * Ordered alphabetically using scriptname.
  * Scriptnames of files in this file should be prefixed with "spell_dh_".
  */
 
+#include "SpellScript.h"
+#include "SpellAuraEffects.h"
+#include "ScriptMgr.h"
+#include "GridNotifiers.h"
+
+enum DemonHunterSpells
+{
+    SPELL_FEL_RUSH_GROUND   = 197922,
+    SPELL_FEL_RUSH_AIR      = 197923,
+    SPELL_FEL_RUSH_DAMAGE   = 192611,
+
+    SPELL_CHAOS_STRIKE_PROC = 193840,
+};
+
+// 197125 - Chaos Strike
+class spell_dh_chaos_strike : public SpellScriptLoader
+{
+    public:
+        spell_dh_chaos_strike() : SpellScriptLoader("spell_dh_chaos_strike") { }
+
+        class spell_dh_chaos_strike_AuraScript : public AuraScript
+        {
+            PrepareAuraScript(spell_dh_chaos_strike_AuraScript);
+
+            void HandleEffectProc(AuraEffect const* aurEff, ProcEventInfo& eventInfo)
```

Comment:
travis-ci build fails on `eventInfo` (unused parameter).
Maybe use `/*eventInfo*/` for now (until fixed)?